app/views/layouts/mailer.html.erb

```diff
@@ -35,8 +35,8 @@
           width: 80%;
       }
       .email-contents button a {
-        text-decoration: none;
-        color: #FFFFFF;
+          text-decoration: none;
```

Comment:
i believe these are duplications.

Reply:
Oh oops! thanks for pointing that out, must have messed up when i pulled in y'alls button stuff

Reply:
Actually its just adding some spacing so I'm not gonna remove it if thats okay since it matches the spacing of the above block